spec/helpers/desks.ts

```diff
@@ -267,6 +269,24 @@ class Desks {
             element(by.model('desk.edit.desk_type')).$('[value="' + deskType + '"]').click();
         };
 
+        /**
+         * Set desk default content template
+         * @param {string} template name
+         **/
+        this.setDeskDefaultContentTemplate = function(template) {
+            element(by.model('desk.edit.default_content_template'))
```

Comment:
Don't use `by.model` for new code. It relies on angular and we'll have to rewrite it when moving to react. It's best to use `data-test-id` to query things since it's not coupled to any framework.